LSP-volar.sublime-settings

```diff
@@ -1,12 +1,12 @@
 {
-	"command": ["${node_bin}", "${server_path}", "--stdio"],
 	"initializationOptions": {
-		// Specify the `typescript.serverPath` to tsserverlibrary.js.
+		// Defines which auto-complete popup trigger characters that Volar defines are allowed.
+		// By default all Volar trigger characters are disabled as ST does a better job.
+		"ignoreTriggerCharacters": [],
```

Comment:
Should this list contain a list of all trigger character that should be ignored?

Reply:
I think it's just badly named. I've tried to clarify the meaning with updated comment